install_config/aggregate_logging.adoc

```diff
@@ -65,9 +63,7 @@ $ oadm new-project logging --node-selector label=value
 +
 [NOTE]
 ====
-The selector in the example is optional. It can be used to specify a
-region or other selector you would like to use to restrict where the
-logging infrastructure deploys.
+Using node selectors is optional. These can be used to specify a region or other selector you would like to use to restrict the nodes onto which the logging infrastructure deploys.
```

Comment:
@tpoitras because of a previous PR from me that recently merged, I can link out to the actual node selector info. I did this instead of trying to explain the dealio.